recipe/meta.yaml

```diff
@@ -21,19 +21,44 @@ requirements:
     - pyproject-metadata 0.7.1
```

Comment:
It seems there should be a different version of `meson` (**>=1.2.3**) for building `meson-python`, see https://github.com/mesonbuild/meson-python/commit/9a7dac0850a6d03b884486f55266cc017cf00ce4